articles/libraries/lock/v10/lock-passwordless-migration-guide.md

```diff
@@ -0,0 +1,162 @@
+
+- `code` if you want to use an Email Code
+- `link` if you want to use a Magic Link
+
+```js
+var options = {
+  oidcConformant: true,
+  passwordlessMethod: 'code' //or link
+};
+var lock = new Auth0LockPasswordless(clientID, domain, options);
+```
+
+### Choose between sms or email
+
+We recommend that you setup which passwordless connections you want enabled in [the dashboard](${manage_url}/#/connections/passwordless), but if you want to have more than one passwordless connection enabled in the dashboard, you can restrict `Auth0LockPasswordless` to use only one of them using the [allowedConnections](/libraries/lock/v10/customization#allowedconnections-array-) option.
+If you have both `sms` and `email` passwordless connections enabled in the dashboard, `Auth0LockPasswordless` will use `email` by default.
+
+#### Example with only sms enabled
+
+```js
+var options = {
+  oidcConformant: true,
+  allowedConnections: ['sms']
+};
+var lock = new Auth0LockPasswordless(clientID, domain, options);
+```
+
+#### Example with only email enabled
+
+```js
+var options = {
+  oidcConformant: true,
+  allowedConnections: ['email']
+};
+var lock = new Auth0LockPasswordless(clientID, domain, options);
+```
+
+### Show the widget
+
+In the previous version, you can call the passwordless method directly (sms, socialOrMagiclink, socialOrSms etc). In the new version, you'll have to use [the show method](/libraries/lock/v10/api#show-) in order to display the widget.
+
+```js
+var options = {
+  oidcConformant: true
+};
+var lock = new Auth0LockPasswordless(clientID, domain, options);
+lock.show();
+```
+
+### Subscribe to events
+
+As of Lock 10, we expose a few events that you can subscribe to in order to be notified when the user is authenticated or an error occurs. So, instead of callbacks from `lock-passwordless`, you have to subscribe to events that you want to know about. To read more about Lock events, see [here](/libraries/lock/v10/api#on-).
+
+```js
+var options = {
+  oidcConformant: true
+};
+var lock = new Auth0LockPasswordless(clientID, domain, options);
+lock.on("authenticated", function(authResult) {
+  alert(authResult.accessToken);
+});
+```
+
+### Customization options
+
+Some options have to be renamed.
+
+* `dict` is now `languageDictionary`. [Read more](https://github.com/auth0/lock#language-dictionary-specification) about the language dictionary specification
+* `connections` is now `allowedConnections`
+* `socialBigButtons` is now `socialButtonStyle`
+* all the authentication options were moved to a new `auth` property. [Read more](https://github.com/auth0/lock#authentication-options)
+
+## Further Reading
+
+::: next-steps
```

Comment:
This looks funny, add the text at the link

![image](https://user-images.githubusercontent.com/11715799/30963239-6377a2c2-a455-11e7-9d18-5e276c8b0b42.png)